grammars/prql-lezer/src/prql.grammar

```diff
@@ -30,6 +30,8 @@ TupleExpression { "{" newline* tupleItem (("," newline*) tupleItem)* ","? newlin
 
 tupleItem { DeclarationTuple | expression | CallExpression | CaseBranch }
 
+Annotation { "@{" commaSep<Declaration>? "}" }
```

Comment:
BTW nice example of `commaSep` being reused! 👏

Reply:
Yeah, `commaSep` is pretty cool, and I felt it might come in handy...